src/renderer/src/helpers/UtilsHelper.ts

```diff
@@ -64,6 +64,14 @@ export class UtilsHelper {
     navigator.clipboard.writeText(text)
   }
 
+  static truncateAddress(text?: string) {
+    if (!text || text.length < 8) return ''
+    const textLenght = text.length
+    const firstPart = text.slice(0, 4)
+    const lastPart = text.slice(textLenght - 4, textLenght)
+    return `${firstPart}...${lastPart}`
+  }
```

Comment:
You don't need this new method, you can use `StringHelper.truncateStringMiddle` instead